chapters/en/slides/module4/module4_01.md

```diff
@@ -0,0 +1,90 @@
+---
+type: slides
+title: 'module4_01'
+---
+
+# Viewing the history of a project
+
+---
+# Viewing the history of a project
+
+There are two ways you can view the Git history of a project:
+
+**In the remote**
+1. On GitHub through the repo's code commit view
+**In your computer**
+2. On Jupyter Lab through the repo's code commit view
+3. On your local machine using `git log`
+
+
+Notes: Do you remember the commit messages that we write for the different changes that we save from the project?
+It is possible to have a look at the history of the full project with any of these 3 different methods
+
+---
+
+# 1. Viewing the history of a project: GitHub
+
+
+<center>
+
+<img src='/module4/vc-history-gb.png' width="90%" alt="404 image"/>
```

Comment:
I know it would be hard to recreate these screenshots, but I think this has been captured on a high-resolution screen and the font sizes are too small. Perhaps you could zoom in and also shrink the browser window when taking this screenshot.